.github/workflows/linux_nightlies.yml

```diff
@@ -0,0 +1,46 @@
+name: Nightly Linux Unit Tests
+
+on:
+  schedule:
+      - cron: '10 * * * *'
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  unit_tests:
+    name: Nightly ${{ matrix.python_version }} ${{matrix.command}}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
Setting `fail-fast` to be false so that we know what all the failures are when they occur.